csaf_2.1/prose/edit/src/distributing.md

```diff
@@ -194,18 +194,25 @@ The index.txt file within MUST provide a list of all filenames of CSAF documents
 
 ### Requirement 13: changes.csv
 
-The file changes.csv MUST contain the filename as well as the value of `/document/tracking/current_release_date` for each
+The file `changes.csv` contains a list of CSAF documents in the current TLP level that were changed recently.
+Therefore, it MUST contain the filename as well as the value of `/document/tracking/current_release_date` for each
 CSAF document in the sub-directories without a heading; lines MUST be sorted by the `current_release_date` timestamp with the latest one first.
+The `changes.csv` SHALL be a valid comma separated values format as defined by [cite](#RFC4180) without double quotes.
+
+> Note: As a consequence of section [sec](#requirement-2-filename) Requirement 2 for filenames and section [sec](#requirement-11-one-folder-per-year)
```

Comment:
@sthagen Should we refer to the titles of these sections or just leave it as it is right now?

If we want to refer to the titles, do you have a special label (e.g. `[title](#ref)`) to do that?

Reply:
I would leave it like that. No one reads only paper version of our work products and all digital user facing delivery items indicate the link target which nicely summarizes the title.